lib/synced/strategies/full.rb

```diff
@@ -92,11 +92,14 @@ def remote_objects_persistor
         lambda do |remote_objects|
           additional_errors_check
           @remote_objects_ids.concat(remote_objects.map(&:id))
+          local_objects_hash = relation_scope.where(@id_key => remote_objects.map(&:id)).each_with_object({}) do |local_object, hash|
```

Comment:
I was wondering why you didnt want to use `remote_objects_ids`, but now I can see its additive collection after each pagination page.
Now I understand, why I saw longer query per each pagination page in the log, as it was building on itself.

Can we assign `remote_objects.map(&:id)` to some informative variable to avoid confusion in the future?
`unprocessed_remote_objects_ids` ?